package.json

```
@@ -35,7 +35,7 @@
   },
   "scripts": {
     "test": "standard && mocha --delay",
-    "prepublushOnly": "npm t",
+    "prepublishOnly": "npm t",
```

```suggestion
    "prepublishOnly": "npm test",
```
no point in abbreviating something nobody has to type :-)